web/app/[locale]/components/waitlist-dialog.tsx

```
@@ -0,0 +1,145 @@
+                <Dialog.Description className="mt-2 text-[15px] text-muted" style={{ lineHeight: 1.5 }}>
+                  {t("description", { platform: platformLabel })}
+                </Dialog.Description>
+
+                <label htmlFor="waitlist-email" className="mt-5 text-sm font-medium">
+                  {t("emailLabel")}
+                </label>
+                <input
+                  id="waitlist-email"
+                  type="email"
+                  autoComplete="email"
+                  autoFocus
```

Biome (2.5.0)

[error] Avoid the autoFocus attribute.

(lint/a11y/noAutofocus)